### DISCUSSION TITLE
Open protocol to implement clients on other platforms?

### DISCUSSION THREAD

**Discussion by mwedmark** [fhdm-dev/scoppy#134] [General]
Is there an open protocol documented over COM-port/network so the community (read me) could implement client in for example Windows?

**Comment by fhdm-dev** [comment]
Hi.
No, the protocol isn't documented.

**Reply by fairderben** [reply]
Thats really sad. A possible implementation into Sigrok would make an absolute workhorse of a tool. Usecases would skyrocket. 
Why is there no interest in keeping this project alive? Its such a well done hardware
